Dockerfile

```diff
@@ -0,0 +1,29 @@
+# gammapy docker container
+FROM continuumio/miniconda3:latest
+RUN apt-get update && apt-get install -y build-essential python3-pip
+
+# define workdir
+WORKDIR /usr/src/app/
+
+# copy content
+COPY . /usr/src/app/
+
+# create conda env
+RUN conda install -c conda-forge mamba
+RUN mamba env create -n gammapy -f environment-dev.yml
+RUN echo 'source activate gammapy' >> ~/.bashrc
```

Comment:
This runs should also be combined into a single run statement and `conda clean --all --yes` called at the end.

Reply:
ok, done ! thanks !